### ISSUE TITLE
make assessment route root route

### ISSUE THREAD

**Issue by mdmartinez** [BettrTogethr/better-together#7]
Thoughts?

**Comment by degoeym** [comment]
No, making it the root route means users will hit that page first and foremost upon going to the main URL for the site.

**Comment by mdmartinez** [comment]
I really meant merging them. Having the assessment functionality on landing is a better UX. It will demo better.
